development/includes/_hero.pug

```diff
@@ -0,0 +1,5 @@
+div.hero
+  img
+  div
+    h1 A Bicycle Day Trip with Pokemon Go
+    a(href="/blog/pokemononabike.html") READ ARTICLE
```

Comment:
This should be "Read Article" (not all caps). Let the css transform the text to all caps.